CMSgov/mint-app
# [EASI-2603] high level project plans article

# EASI-2603

## Changes and Description

- Add new `HighLevelProjectPlan` index and table 



## How to test this change

1. Navigate to help and knowledge center
2. click `View getting started articles`
3. Click `High-level project plans`
4. Ensure all links work and design matches figma


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
